tests/matrix_free/parallel_multigrid_fullydistributed.with_mpi=true.with_p4est=true.mpirun=7.output

```diff
@@ -0,0 +1,289 @@
+
```

Comment:
Is the 7 MPI rank case really necessary or is it enough to run it with 3 ranks? I'm asking because I would be conservative with tests on more than 5 MPI ranks or so. We have a few dozen, but should think a second before we add more, especially since the `parallel_multigrid_...` are quite heavy.

Reply:
I have removed this test. I also have reduced the number of needed processes in the other test. There are 2 p:f:t tests left with more than 5 processes: one uses 9 processes (however is very cheap, since I am testing the corner case that there a processes that do not get any cells) and one uses 7 (to test the construction of `ConstructionData` in groups of size 3+3+1). I would like to keep these.